peachjam/views/legislation.py

```diff
@@ -54,13 +54,13 @@ def get_notices(self):
 
         points_in_time = self.get_points_in_time()
         work_amendments = self.get_work_amendments()
-
         current_object_date = self.object.date.strftime("%Y-%m-%d")
-        work_amendments_dates = [
-            work_amendment["date"] for work_amendment in work_amendments
-        ]
 
-        latest_amendment_date = max(work_amendments_dates)
+        if work_amendments:
+            work_amendments_dates = [
+                work_amendment["date"] for work_amendment in work_amendments
+            ]
+            latest_amendment_date = max(work_amendments_dates)
```

Comment:
`latest_amendment_date` now won't be declared for use in the if statement below. It should default to None if there are no amendments.